js/modules/cohortbuilder/components/CohortExpressionEditorTemplate.html

```diff
@@ -220,4 +220,18 @@
 			</ul>
 		</div>
 	</div>
+
+	<div role="tabpanel" data-bind="css: { active: $component.expressionMode() == 'censorWindow' || $component.expressionMode() == 'all'}" class="tab-pane">
+		<div class="heading">
+			<b>Cohort Censor Window</b>
+			Drops all persons, who do no fall inside the censor window.
+		</div>
+		<div style="padding: 5px 0px">
+			<ul>
+				<li>
+					<period-input params="Period: expression().CensorWindow"></period-input>
+				</li>
+			</ul>
+		</div>
+	</div>
```

Comment:
This looks like it always is always showing the censor window:  I think we'd want to have a button that when CensorWindow == null it shows and says 'Set Censor Window' and clicking it adds a new Period() to the CensorWindow.  (which will hide the button and show the UI).  Elsewhere, there would be a 'Remove censor window' which would set CensorWindow  to null, which would update the UI to show the 'Add Censor Window button' etc.

However, @pavgra , the main problem is that you're working off of a out of date version of the CohortExpressionEditor component. This was drastically redone in the 2.3 branch.  @anthonysena  or @fdefalco should give you guidance on how you should apply those changes to that feature branch.

Reply:
@chrisknoll, so which branch should I be branching off?

Reply:
@anthonysena or @fdefalco  should be able to provide you with that.